cmd/kube_create.go

```diff
@@ -30,43 +30,69 @@ type kubeBootstrapStep struct {
 type kubeCluster struct {
 	Name    string
 	Version string
+	Address string
 }
 
 // kubeBootstrapSteps represents a k8s instance bootstrap steps
 var kubeBootstrapSteps = []kubeBootstrapStep{
 	{name: "Instance system upgrade", command: `\
-sudo apt-get update && sudo apt-get upgrade -y && \
-nohup sh -c 'sleep 5s ; sudo reboot' & \
-exit`},
-	{name: "Docker Engine installation", command: `\
+sudo apt-get update && sudo apt-get upgrade -y
 sudo apt-get install -y \
     apt-transport-https \
     ca-certificates \
 	curl \
 	software-properties-common
-
+nohup sh -c 'sleep 5s ; sudo reboot' &
+exit`},
+	{name: "Docker Engine installation", command: `\
 curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-
 sudo add-apt-repository \
    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
    $(lsb_release -cs) \
    stable"
-
 sudo apt-get update && sudo apt-get install -y docker-ce=18.06.0~ce~3-0~ubuntu
```

Comment:
also https://packages.ubuntu.com/cosmic/docker.io

Reply:
at the end of the day, it's harder to maintain on our side to do so.

Reply:
We're not exactly talking about 1000+ or even 100+ LoC to maintain here... Let's agree to disagree 😏